dsc_lib/src/functions/target_path.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::DscError;
+use crate::configure::context::Context;
+use crate::functions::{AcceptedArgKind, Function};
+use serde_json::Value;
+use tracing::debug;
+
+#[derive(Debug, Default)]
+pub struct TargetPath {}
+
+/// Implements the 'mountedpath' function.
```

Comment:
```suggestion
/// Implements the 'TargetPath' function.
```
there's a few other references to mounted path in this file and others